source/releasenotes/2025-05-07-terminus-4-0-0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Terminus 4.0.0 release now available"
+published_date: "2025-05-07"
+categories: [tools-apis]
+---
+
+Terminus [4.0.0](https://github.com/pantheon-systems/terminus/releases/tag/4.0.0) is now available.
+The next generation of Terminus adds support for PHP 8.4 and removes support for PHP versions 8.1 and below.
```

Comment:
```suggestion
This release adds support for PHP 8.4 and removes support for PHP versions 8.1 and below.
```

We're about to use the word "generation" to mean something different as we do a more complicated and intertwined update to container infra. We should not use it here when we mean something much simpler.

Reply:
```suggestion
This release adds support for PHP 8.4 and removes support for PHP versions 8.1 and below.
Terminus 4 is compatible with PHP 8.2 and 8.3.
See [Supported Terminus and PHP versions](/terminus/supported-terminus).
The PHP version refers to the local machine on which Terminus itself runs (e.g. developers' own computers or containers for continuous integration). This is distinct from PHP versions that run websites on Pantheon's infrastructure (which still include support for PHP 7.x).
```